src/WWT.Providers/FilePathOptions.cs

```diff
@@ -14,12 +14,15 @@ public static FilePathOptions CreateFromConfig()
                 WWTDEMDir = ConfigurationManager.AppSettings["WWTDEMDir"],
                 WwtTilesDir = ConfigurationManager.AppSettings["WWTTilesDir"],
                 WwtGalexDir = ConfigurationManager.AppSettings["WWTGALEXDIR"],
+                WwtWebDir = ConfigurationManager.AppSettings["WWTWEBDIR"],
```

Comment:
In the current VMs, the `WWTWEBDIR` variable is `c:\wwt\1.0`, which is the base directory that the web app is being served out of (the directory containing `Global.asax`, etc.). So I'd guess that instead of turning this into an AppSetting, there is probably a better way to compute such paths? In this PR, the variable is only used to locate the `colors.map` file, which indeed currently lives in the repo at `src/WWTMVC5/WWTWeb/colors.map`.

As a side note, the `mandel.aspx` endpoint is super obscure and rarely used, so it is not worth engineering something fancy to cover this particular use case.

Reply:
Ah good point. There's a method on the context object that handles that